src/app/shared/comment-editor/comment-editor.component.ts

```diff
@@ -69,6 +68,56 @@ export class CommentEditorComponent implements OnInit {
     this.commentField.setValidators([Validators.maxLength(this.maxLength)]);
   }
 
+  onKeyPress(event) {
```

Comment:
This is a 'long method' and might be better broken up if possible into multiple methods

Reply:
@gycgabriel, good point, I agree with breaking this up, have already done so, do take a look!